tests/std/tests/P0528R3_cmpxchg_pad/test.cpp

```diff
@@ -53,8 +70,35 @@ struct alignas(4) X4 {
     void set(char v) {
         x = v;
     }
+
+    bool check(const char v) const {
+        return x == v;
+    }
+};
+#pragma warning(pop)
+
+#pragma pack(push)

```

Comment:
Why is packing being pushed/popped here, when it's apparently not being altered?